### PR TITLE
CONFIG : Dotnet Version Update

### PR DESCRIPTION
**Pull request by tharaniannamalai**

### CODE REVIEW

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -9,13 +9,14 @@ on:
 jobs:
   build:
     runs-on: windows-2019
+    timeout-minutes: 0
```
Comment (ShriHumrudha) [inline .github/workflows/dotnet.yml:12]: This should not appear. We need to fix it.

File: Random.Services.Infrastructure.Build/Program.cs [hunk 2]
```diff
@@ -52,7 +52,7 @@ static void Main(string[] args)
 
                                 TargetDotNetVersion = new TargetDotNetVersion
                                 {
-                                    DotNetVersion = "6.0.101",
+                                    DotNetVersion = "7.0.0",
```
Comment (ShriHumrudha) [inline Random.Services.Infrastructure.Build/Program.cs:55]: This should be 7.0.0-preview.1

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: Random.Services.Infrastructure.Build/Program.cs (hunk 2 above)
Comment (tharanisantosh) [reply Random.Services.Infrastructure.Build/Program.cs:55]: Comment Resolved.

File: .github/workflows/dotnet.yml (hunk 1 above)
Comment (tharaniannamalai) [reply .github/workflows/dotnet.yml:12]: Comment Resolved.

**Comment by ShriHumrudha** [comment]
@tharanisantosh refer this PR - 
https://github.com/Mums-Who-Code/JustLearn.Services/pull/38
https://github.com/Mums-Who-Code/JustLearn.Services/pull/39

**Pull request closed without merging**